node/cn/gasprice/feehistory.go

```diff
@@ -228,6 +245,10 @@ func (oracle *Oracle) FeeHistory(
 		next    = oldestBlock
 		results = make(chan *blockFees, blocks)
 	)
+	percentileKey := make([]byte, 8*len(rewardPercentiles))
+	for i, p := range rewardPercentiles {
+		binary.LittleEndian.PutUint64(percentileKey[i*8:(i+1)*8], math.Float64bits(p))
+	}
 	for i := 0; i < maxBlockFetchers && i < blocks; i++ {
```

Comment:
Is there any reason to keep `maxBlockFetchers == 1` which is `4` in Ethereum?

Reply:
I don't see any. I've increased `maxBlockFetchers` to `4`